genapkovl-lima.sh

```diff
@@ -32,6 +32,10 @@ makefile root:root 0644 "$tmp"/etc/hostname <<EOF
 $HOSTNAME
 EOF
 
+if [ "$LIMA_CGROUP_MODE" != "hybrid" ]; then
+    sed -E "s/#(rc_cgroup_mode).*/\1=\"$LIMA_CGROUP_MODE\"/" /etc/rc.conf >"$tmp"/etc/rc.conf
```

Comment:
Wouldn't there technically be an issue if the builder and the build target uses different versions? (I don't think that's actually possible, though, so we should be fine… but I didn't spot any other copy-from-host things in this file at a quick glance.)

Reply:
I wondered about this too, but I don't know where also to get a matching copy of `rc.conf`. Lmk if you have any idea!

Most lines in the default `rc.conf` are just commented out defaults anyways, so not sure if this matters a lot:

```console
# grep -E -v '^(#|$)' /etc/rc.conf
rc_tty_number=12
respawn_delay=2
respawn_max=5
respawn_period=1800
```